app-authz-photoz/photoz-testsuite/pom.xml

```diff
@@ -25,7 +25,7 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
     <artifactId>photoz-testsuite</artifactId>
-    <packaging>jar</packaging>
+    <packaging>pom</packaging>
```

Comment:
Out of curiosity - why is this change here?

Reply:
This is a project with only tests, and having it packaged as a JAR will make Maven complain with "Warning:  JAR will be empty - no content was marked for inclusion!"

So marking it as a POM will get rid of that warning.